packages/core/src/actions/smartAccount/sendTransaction.ts

```diff
@@ -62,6 +62,7 @@ export async function sendTransaction<
     account: account as SmartContractAccount,
     uoStruct,
     context,
+    overrides,
```

Comment:
why do we need overrides passed into the internal _senduserop? Do we use it there? or is this just passed in because overrides are available and there is no reason not to?